File: src/FizzBuzz/fizz_buzz.hs
module Divisor.Main where

fizz_buzz :: Int -> IO ()
fizz_buzz n
    | mod n 3 == 0 && mod n 5 == 0 = putStrLn "FizzBuzz"
    | mod n 3 == 0 = putStrLn "Fizz"
    | mod n 5 == 0 = putStrLn "Buzz"
    | otherwise = putStrLn (show n) 

main :: IO ()
main = mapM_ fizz_buzz [1..100]

File: src/FizzBuzz/fizz_buzz.php
<?php
for ($i = 0; $i < 100; $i++) {
    $out = null;
    if ($i % 3 === 0) {
        $out = 'Fizz';
    }
    if ($i % 5 === 0) {
        $out .= 'Buzz';
    }
    $out = $out ?? $i;
    echo "$out\n";
}
;